src/app/internarbeidsflatedecorator/useHandleQueryParams.ts

```diff
@@ -0,0 +1,44 @@
+import { useHistory } from 'react-router';
+import { useQueryParams } from '../../utils/urlUtils';
+import { paths, setNyBrukerIPath } from '../routes/routing';
+import { useOnMount } from '../../utils/customHooks';
+import { loggEvent } from '../../utils/frontendLogger';
+import { INFOTABS } from '../personside/infotabs/InfoTabEnum';
+import { useDispatch } from 'react-redux';
+import { usePostResource } from '../../rest/consumer/usePostResource';
+
+function useHandleQueryParams() {
+    const history = useHistory();
+    const queryParams = useQueryParams<{ sokFnr?: string; oppgaveid?: string; behandlingsid?: string }>();
+    const sokFnr = queryParams.behandlingsid ? undefined : queryParams.sokFnr === '0' ? '' : queryParams.sokFnr;
+    const plukkOppgaverResource = usePostResource(resources => resources.plukkNyeOppgaver);
+    const dispatch = useDispatch();
+
+    useOnMount(() => {
+        if (queryParams.behandlingsid && queryParams.sokFnr && queryParams.oppgaveid) {
```

Comment:
Har vi dobbeltsjekket at `behandlingsid`, `sokFnr` og `oppgaveid` alle vil være satt når man kommer fra gosys?

Reply:
Nei, egentlig ikke. Trenger hjelp av folk som skjønner litt mer gosys og premisser for selve operasjonen :) regnet med at det er alltid vil være behandlingsid og oppgaveid ettersom du kommer fra en oppgave i gosys